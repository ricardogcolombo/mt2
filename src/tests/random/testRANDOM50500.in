50 500
1 39 69 31 106
1 38 30 9 40
1 26 118 2 44
1 42 59 9 103
1 43 33 44 118
1 10 110 33 10
1 45 53 29 98
1 21 130 1 4
1 10 4 35 92
1 13 4 27 130
1 30 9 12 43
1 2 34 36 4
1 2 94 49 39
1 33 39 47 62
1 23 42 49 38
1 13 113 36 13
1 18 124 31 62
1 38 47 18 23
1 22 30 32 32
1 31 22 20 8
1 14 14 30 72
1 21 90 4 25
1 13 41 48 54
1 13 83 19 112
1 49 125 8 4
1 7 91 39 81
1 45 5 30 42
1 29 64 50 90
1 5 15 15 22
1 10 112 29 35
1 5 119 19 86
1 9 6 35 96
1 10 29 26 93
1 5 111 24 50
1 37 60 15 90
1 32 56 2 46
1 49 16 28 119
1 41 129 47 83
1 15 16 44 9
1 33 40 43 16
1 16 43 36 96
1 5 127 39 126
1 39 74 11 128
1 2 88 9 76
1 13 3 7 94
1 43 60 6 122
1 33 85 29 55
1 3 47 33 8
1 43 97 39 87
1 24 76 30 27
1 15 29 1 79
1 41 22 17 18
1 12 30 1 98
1 18 114 38 74
1 34 103 11 85
1 38 5 48 80
1 12 51 15 14
1 7 30 45 58
1 7 19 32 11
1 4 96 3 124
1 21 29 26 27
1 12 65 29 81
1 44 17 11 127
1 41 8 1 97
1 7 98 5 1
1 46 70 25 41
1 20 54 1 21
1 14 67 2 123
1 33 97 26 77
1 42 44 23 18
1 40 53 33 72
1 2 25 32 56
1 24 26 21 54
1 32 40 11 70
1 33 27 40 56
1 47 23 38 34
1 37 118 28 6
1 23 88 33 33
1 14 49 39 96
1 12 92 13 93
1 12 70 41 76
1 16 10 27 49
1 30 97 23 76
1 21 43 2 50
1 1 3 39 58
1 3 79 24 36
1 39 34 6 108
1 39 73 28 36
1 25 9 47 38
1 2 32 33 77
1 32 59 47 19
1 4 27 5 26
1 43 67 37 17
1 10 5 44 24
1 4 53 50 66
1 49 70 9 75
1 38 59 30 17
1 40 36 8 102
1 47 107 28 120
1 2 130 26 67
1 42 16 13 121
1 47 72 41 73
1 45 25 13 112
1 37 67 5 126
1 14 101 46 38
1 42 103 30 82
1 23 117 11 2
1 49 114 12 47
1 24 52 15 105
1 1 21 36 94
1 11 5 17 85
1 14 28 3 15
1 6 100 3 16
1 28 129 39 121
1 16 15 39 36
1 37 120 5 82
1 3 87 12 100
1 38 29 32 30
1 40 38 45 42
1 47 85 30 101
1 48 121 29 72
1 16 119 39 47
1 13 130 35 37
1 23 92 38 75
1 38 14 4 110
1 12 31 4 55
1 16 84 15 9
1 22 6 30 109
1 19 39 35 15
1 49 76 25 77
1 19 10 37 112
1 16 123 9 28
1 34 28 26 4
1 27 94 34 21
1 42 13 6 121
1 42 97 24 5
1 17 53 49 95
1 3 101 12 31
1 14 33 13 51
1 24 92 14 62
1 30 59 9 66
1 15 27 8 21
1 38 49 40 74
1 28 107 17 65
1 22 83 6 44
1 25 117 44 79
1 40 83 26 81
1 25 108 29 71
1 5 105 15 123
1 21 78 20 85
1 16 94 8 118
1 18 6 39 9
1 12 4 23 52
1 37 10 38 78
1 31 54 24 61
1 16 70 24 88
1 29 51 14 38
1 45 45 43 29
1 38 130 13 106
1 46 12 31 127
1 11 81 12 28
1 49 90 6 23
1 42 56 39 63
1 43 110 4 15
1 47 119 36 81
1 50 130 15 7
1 35 25 41 78
1 9 3 45 86
1 19 47 7 128
1 30 53 39 116
1 3 94 26 98
1 15 96 25 98
1 8 76 9 117
1 4 14 29 123
1 24 94 6 118
1 14 16 3 103
1 26 49 13 3
1 2 57 26 54
1 43 36 44 67
1 22 18 8 56
1 46 35 15 10
1 45 73 25 75
1 22 128 7 85
1 40 91 16 33
1 28 59 9 91
1 13 119 43 72
1 23 45 34 15
1 12 80 3 93
1 30 53 5 13
1 50 74 23 90
1 18 127 12 83
1 25 72 41 25
1 19 92 30 117
1 43 67 25 128
1 15 102 31 66
1 35 78 4 115
1 42 34 38 93
1 1 126 17 97
1 44 58 43 70
1 5 7 16 54
1 45 41 5 71
1 32 35 1 101
1 36 48 23 62
1 20 117 42 6
1 44 94 38 11
1 48 37 41 44
1 46 4 19 57
1 36 91 25 74
1 4 106 21 15
1 31 96 17 9
1 22 8 42 33
1 4 82 39 116
1 30 7 26 58
1 21 24 48 121
1 23 16 6 96
1 36 5 47 44
1 41 101 1 90
1 42 37 49 83
1 39 105 10 29
1 38 82 50 17
1 39 31 16 59
1 36 27 34 69
1 1 89 26 126
1 32 50 21 13
1 28 77 25 84
1 20 46 18 11
1 29 101 7 126
1 6 53 40 121
1 1 119 22 20
1 4 29 46 126
1 27 48 6 67
1 47 4 41 98
1 41 78 14 73
1 22 117 6 60
1 24 18 45 69
1 45 39 38 72
1 18 54 17 76
1 9 101 42 109
1 7 108 26 49
1 25 97 40 94
1 20 86 2 70
1 16 69 22 61
1 42 55 15 21
1 25 73 32 50
1 22 130 21 90
1 26 112 19 53
1 43 76 18 63
1 21 59 14 93
1 34 33 21 28
1 42 96 45 48
1 37 12 13 18
1 24 98 48 127
1 7 110 20 101
1 42 90 26 97
1 35 47 42 62
1 32 29 46 125
1 45 72 31 6
1 42 97 40 87
1 40 116 33 111
1 3 86 13 124
1 2 48 28 32
1 25 52 27 65
1 18 102 23 82
1 4 88 38 124
1 19 65 36 21
1 22 44 17 126
1 22 100 34 81
1 24 88 45 10
1 12 125 48 86
1 30 90 12 26
1 43 95 16 103
1 20 90 38 22
1 38 99 15 100
1 35 130 25 0
1 49 24 23 9
1 13 22 10 123
1 26 83 6 50
1 9 4 30 124
1 31 118 49 103
1 29 115 2 113
1 43 103 26 76
1 35 118 32 82
1 4 110 16 43
1 12 5 43 84
1 30 0 23 123
1 41 80 20 67
1 23 79 1 101
1 35 101 10 31
1 49 68 23 86
1 11 76 4 81
1 21 43 31 23
1 49 11 23 35
1 12 110 13 38
1 30 95 42 117
1 43 105 12 49
1 8 76 35 87
1 28 59 31 70
1 14 4 5 24
1 34 120 18 85
1 48 34 7 94
1 3 102 8 98
1 35 10 2 69
1 8 130 15 55
1 38 20 37 46
1 41 38 40 39
1 22 37 23 78
1 9 94 48 53
1 44 73 41 88
1 35 21 14 22
1 13 75 42 80
1 8 29 29 20
1 3 80 5 22
1 11 99 1 3
1 2 122 9 18
1 33 37 47 30
1 46 27 38 34
1 30 71 50 123
1 46 18 2 46
1 38 42 19 18
1 1 112 12 32
1 39 122 2 50
1 1 49 26 6
1 4 71 11 31
1 16 77 47 9
1 27 126 23 13
1 4 57 10 64
1 15 13 36 54
1 18 53 27 69
1 34 89 29 38
1 12 63 2 83
1 14 128 24 76
1 43 6 39 46
1 1 26 7 56
1 23 62 20 114
1 10 49 40 111
1 8 16 49 25
1 41 25 33 77
1 32 46 11 18
1 33 20 13 117
1 41 100 1 30
1 21 100 40 12
1 25 92 5 86
1 31 95 28 30
1 43 27 36 109
1 27 120 43 72
1 38 11 4 110
1 22 89 5 109
1 2 38 29 104
1 24 101 38 46
1 21 14 1 68
1 9 72 4 30
1 19 75 1 94
1 17 70 31 67
1 4 85 8 27
1 19 8 14 16
1 36 38 46 1
1 33 116 19 94
1 33 37 4 54
1 40 110 24 85
1 16 6 25 32
1 24 74 41 42
1 6 116 36 76
1 36 11 32 88
1 36 78 21 30
1 40 106 4 3
1 13 41 19 82
1 47 65 8 48
1 28 51 23 112
1 3 66 7 90
1 22 26 33 110
1 37 73 50 17
1 8 93 15 61
1 5 30 39 69
1 30 31 35 85
1 10 88 32 106
1 21 123 15 24
1 31 130 28 123
1 7 24 20 2
1 16 19 45 63
1 31 85 18 75
1 36 86 45 125
1 29 55 22 101
1 45 42 30 104
1 35 122 14 102
1 28 16 39 5
1 5 113 36 97
1 49 21 14 97
1 21 108 38 114
1 33 32 36 10
1 39 51 19 72
1 2 28 43 47
1 1 74 42 60
1 4 8 48 7
1 20 81 15 129
1 19 67 27 1
1 24 43 26 34
1 46 84 17 99
1 37 68 10 2
1 34 89 41 69
1 3 78 28 22
1 19 43 50 85
1 30 126 29 25
1 33 36 43 74
1 38 3 50 19
1 49 113 9 78
1 32 66 3 26
1 17 89 18 16
1 47 59 41 111
1 13 17 15 6
1 43 53 28 90
1 6 44 42 9
1 48 11 16 56
1 32 42 44 89
1 14 92 33 22
1 45 21 32 64
1 12 5 41 54
1 27 2 14 10
1 32 71 11 112
1 23 78 33 119
1 9 58 37 49
1 18 47 30 40
1 20 99 4 42
1 34 87 12 116
1 31 82 2 53
1 24 4 1 126
1 40 44 3 33
1 2 17 24 55
1 15 93 41 102
1 6 97 9 74
1 1 117 17 62
1 34 38 38 19
1 19 47 36 54
1 1 66 45 32
1 14 121 26 105
1 24 28 45 127
1 20 122 44 51
1 11 110 23 89
1 37 86 23 127
1 16 91 47 8
1 39 127 30 89
1 3 30 48 127
1 3 10 48 83
1 45 108 30 99
1 38 106 49 5
1 33 9 19 35
1 20 65 17 127
1 32 0 7 11
1 31 15 47 101
1 1 130 49 125
1 6 100 39 44
1 9 88 22 26
1 1 10 11 96
1 42 109 30 59
1 29 104 12 116
1 8 48 16 78
1 44 42 36 108
1 50 68 1 20
1 39 112 30 76
1 30 96 50 89
1 8 73 15 27
1 47 18 1 59
1 40 48 23 67
1 44 88 11 107
1 18 78 19 62
1 22 3 9 89
1 30 84 28 115
1 2 50 31 8
1 1 67 4 84
1 22 113 18 34
1 26 39 43 123
1 34 84 14 18
1 41 16 42 85
1 5 35 24 111
1 42 16 32 73
1 39 105 18 71
1 21 20 19 24
1 27 125 1 3
1 37 70 32 13
1 13 25 1 126
1 3 102 42 125
1 38 80 9 112
1 25 56 1 79
1 50 66 38 10
1 50 62 21 70
1 29 129 6 49
1 21 119 12 109
1 44 73 45 88
1 21 51 40 52
1 21 97 26 17
1 43 114 34 130
1 31 129 34 109
1 29 79 47 69
1 14 51 41 9
1 7 32 31 19
1 15 119 3 129
1 32 53 31 88
1 22 86 19 48
1 47 126 7 27
1 24 35 31 47
1 5 71 35 60
