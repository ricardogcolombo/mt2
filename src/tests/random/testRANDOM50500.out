0.603293
0.515744
0.459394
0.504433
0.442252
0.639715
0.791988
0.405901
0.548448
0.345799
0.605895
0.291727
0.478946
0.492292
0.362959
0.538776
0.545705
0.381726
0.407894
0.624297
0.443117
0.435742
0.597806
0.447284
0.638553
0.429089
0.719892
0.438377
0.422863
0.404294
0.573601
0.538038
0.486543
0.636469
0.486378
0.422815
0.466580
0.429491
0.546974
0.617886
0.630791
0.563326
0.443128
0.377081
0.499384
0.379348
0.393901
0.529777
0.453893
0.560400
